src/vehicle.h

```diff
@@ -858,6 +858,23 @@ class vehicle : public JsonSerializer, public JsonDeserializer
 
     /*@}*/
 
+    /**
+     * If target for the turret is set, it returns the set target and unsets target.
+     * Otherwise finds a target and returns it.
+     * @return A target for the turret or turret's position if no target is found.
+     */
+    tripoint turret_acquire_target( vehicle_part &pt );
+    /**
+     * Fires a loaded turret at the target.
+     * @return Number of shots fired.
+     */
+    int turret_fire( vehicle_part &pt, const tripoint &targ );
+
+    void turret_reload( vehicle_part &pt );
```

Comment:
These are an implementation detail (and one that's rapidly becoming defunct in a branch I'm working on). There should be no need to expose them and indeed doing so suggests your unit test is testing the implementation as opposed to the API